### PR TITLE
clean up makefile

### PR DESCRIPTION
**Pull request by shrik3**

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -1,62 +1,55 @@
...
+
 
 .PHONY: all release debug clean install qvisor_release qvisor_debug cuda_make cuda_all
 
 all:: release debug
 
 cuda_all:: cuda_release cuda_debug
 
-release:: qvisor_release qkernel_release 
+release:: qvisor_release qkernel_release $(VDSO)
```
[the remark this answers is not in the document]
Comment (shrik3) [reply makefile:31]: I didn't add them and afaik it doesn't matter for now.

https://stackoverflow.com/questions/7891097/what-are-double-colon-rules-in-a-makefile-for

File: makefile [hunk 2]
```diff
@@ -1,62 +1,55 @@
...
+
+debug:: qvisor_debug qkernel_debug vdso $(VDSO)
```
Comment (chl337) [inline makefile:33]: Is the "vdso $(VDSO)" intentional?

Comment (shrik3) [reply makefile:33]: no, I'll remove it, thanks.

The idea was to explicitly require the vdso build target (`vdso/vdso.so`) so it only needs to be built once.

**Review by chl337 — APPROVED** [review]

**Comment by chengchen666** [comment]
Not familiar with makefile, but LGTM.

**Review by chengchen666 — APPROVED** [review]

**Comment by shrik3** [comment]
> Not familiar with makefile, but LGTM.

as long as I'm not breaking the CUDA build. I can't test it on my machine.

**Pull request merged**